can_tools/scrapers/__init__.py

```diff
@@ -1,10 +1,6 @@
-from .base import CMU, DatasetBaseNeedsDate, DatasetBaseNoDate, InsertWithTempTable
-
+from .base import (CMU, DatasetBaseNeedsDate, DatasetBaseNoDate,
```

Comment:
👋  just saw this PR and dropping in with a couple of style questions haha.  Thoughts on using absolute path names? the [google style guide](https://google.github.io/styleguide/pyguide.html#23-packages) recommends them and I think it helps reorganization.  (for instance if you move a file don't have to change all of it's imports)

Reply:
Also if we want to go that direction, I'm happy to go through and change all of the existing imports myself

Reply:
Happy to use absolute path names.

I think I've seen different opinions on this, but overall I find absolute imports to be easier also

Reply:
I actually prefer relative imports -- I admit that I'm not sure exactly why I prefer them though...

Happy to do absolute imports if the two of you like it better (especially if @sglyon or @ghop02 does the work to update everything we've written so far :laughing: )

Reply:
moved to absolute.